examples/wallet/app/utils/mnemonic.js

```diff
@@ -0,0 +1,20 @@
+import { validateMnemonic, mnemonicToSeedSync } from 'bip39';
+
+import wordlist from './wordlist.en';
+
+export const isValidMnemonic = (mnemonicPhrase: string, numberOfWords = 12) => {
+  return (
+    mnemonicPhrase &&
+    mnemonicPhrase.split(' ').length === numberOfWords &&
+    validateMnemonic(mnemonicPhrase, wordlist)
+  );
+};
+
+export const createSeedFromMnemonic = (
+  mnemonicPhrase: string,
+  mnemonicPassword?: string
+) => {
+  const password = mnemonicPassword || '';
+  const seed = mnemonicToSeedSync(mnemonicPhrase, password);
```

Comment:
⚠️ PLEASE DO NOT DO THIS ⚠️ 

This is the WRONG function to generic a seed from a mnemonic! The seed generated by this function is not the same as the one Cardano implements (the algorithm is the same but parameters are different and the `bip39` library does NOT expose the parameters you need to generate the correct seed)

You have two options:

A) you fork the bip39 library and reverse-engineer the parameters needed from the Rust codebase (be careful if you do this because if I remember correctly some Rust crypto parameters are swapped compared to the Javascript bip39 crypto library).

B) You accept this PR https://github.com/input-output-hk/js-chain-libs/pull/58 and I've already tested the key is correct

If you don't believe me, you can see it for yourself that the results are different (it's also very clear if you look compare the source code for bip39 vs the Rust source code) https://repl.it/repls/ItchyIrritatingRuntimes

Reply:
this is an example wallet, it's at the moment meant to demonstrate key feature of the blockchain and interact with it, not strong/standard/ wallet primitives.

Reply:
Well I'm scared people will look at this code as a reference and accidentally use this in a production wallet. @dmernies-iohk can we at least put a comment warning that this is not how seed generation works in Cardano and to refer to this document for the standard?

https://github.com/satoshilabs/slips/blob/master/slip-0023.md